src/components/App.js

```diff
@@ -1,9 +1,14 @@
 import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
 
-function App() {
+const App = () => { // eslint-disable-line
   return (
-    <div className="App" />
+    <div className="App">
+      <Display />
+      <ButtonPanel />
```

Comment:
- [ ] components should be wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))